js/main.js

```diff
@@ -59,4 +59,8 @@
     });
 
     new DawUI().call();
+
+    if(document.location.host.indexOf('localhost')>=0){
```

Comment:
doesn't this mean it *only* works on localhost domains? We want the opposite behaviour (e.g. it only works on non-localhost domains) right?